03-altalanos.md

```diff
@@ -7,7 +7,7 @@
 
 2.  A versenyzők a bíró hívására a szabályos felszerelésben, vívásra készen jelennek meg a küzdőtérnél, minden egyes körmérkőzés vagy egyenes kieséses mérkőzés előtt.
 
-3.  A bíró minden forduló előtt ellenőrzi a vívók felszerelését, és meggyőződik a biztonsági felszerelések és a fegyverek megfelelő állapotáról. A bíró bármikor megismételheti felszerelés ellenőrzését, amikor szükségesnek ítéli. A nem megfelelő felszereléssel megjelenő versenyzők a szabálykönyv rendelkezései szerinti büntetéstben részesülnek, és nem kezdhetik meg vagy folytathatják a versenyzést, amíg a kérdéses felszerelésre a fegyverellenőrzésen jóváhagyást nem szereznek (lásd a felszerelés szabályokat), vagy le nem cserélik jóváhagyottra. 
```

Comment:
Itt csak a bíróról van szó aki a pool elején és minden DE meccs előtt megnézi, hogy rajta van-e a fegyverellenőrző jel a cuccokon, amin kell, sisakon a FIE jel (ha kötelező az 1600N), és rajtuk van-e az előírt felszerelés, ránézésre oké-e, kukivédő cicivédő stb. 

Azt szerintem a felszerelés szabályokhoz írjuk, hogy mindnki köteles fegyverellenőrzésre menni a verseny megkezdése előtt. (ha még nincs ott, azt nem néztem) 


Ide pontosításnak meg beírhatnánk ilyesmit, hogy a bíró ellenőrzi az érintett fegyvereken és felszerelésen a fegyverellenőrzés jóváhagyó jelét, a FIE standard jelzést, és hogy a felszerelés nem hiányos.